src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -113,6 +114,7 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         const isCellEditable = isCellHovered && !!getProp(metadataColumn, 'canEdit', false);
         const item = items[rowIndex - 1];
         const { name } = item;
+        const fields = getProp(item, 'metadata.fields', []);
```

Comment:
Can you add a constant for this string as well?

Reply:
This is object path `item.metadata.fields`. Don't think we should create constant out of this as this is not going to be used elsewhere?